### ISSUE TITLE
Clarify 'Coverages' section in SDWBP

### ISSUE THREAD

**Issue by dr-shorthair** [w3c/sdw#1265]
The Coverages section is very abstract and does not use the 99% case for coverages - i.e. gridded data - to introduce the concept for new readers. Also, no real examples (or pictures).

**Comment by chris-little** [comment]
@dr-shorthair Agreed, and the statement `As defined by [ISO-19123] it is simply a data structure...` is too naive, as ISO19123, and its forthcoming update, both emphasise operations/methods on the data structure too: `CV_Coverage  shall  support  three  attributes,  five  operations,  and  three associations`
